utils/migration_tools/from_legacy_neptune/README.md

```diff
@@ -0,0 +1,146 @@
+# Copying runs from Neptune Legacy (2.x) to Neptune (3.x)
+
+This script helps you copy runs from Neptune Legacy (2.x) to Neptune (3.x).
+
+---
+## Changelog
+- 2025-05-19 - Initial release
+
+---
+
+## Prerequisites
+- A Neptune 2.x workspace with runs to migrate.
+- A Neptune 3.x workspace with write access to migrate the runs to.
+- Latest versions of the `neptune-scale` and `neptune` Python packages installed:
+  ```bash
+  pip install -U neptune-scale neptune
+  ```
+
+---
+
+## Quick Start
+
+```bash
+python runs_migrator.py \
+  --legacy-token <NEPTUNE_2.X_API_TOKEN> \
+  --new-token <NEPTUNE_3.X_API_TOKEN> \
+  --new-workspace <NEPTUNE_3.X_WORKSPACE_NAME> \
+  --legacy-project <LEGACY_WORKSPACE/PROJECT_NAME> \
+```
+
+---
+
+## Arguments
+
+| Argument | Required | Description |
+| --- | --- | --- |
+| `--legacy-token` | Yes | API token for the legacy Neptune workspace (2.x). |
+| `--new-token` | Yes | API token for the new Neptune workspace (3.x). |
+| `--legacy-project` | Yes | Name of the legacy project in the format `WORKSPACE_NAME/PROJECT_NAME`. |
+| `-w`, `--new-workspace` | Yes | Name of the new workspace in Neptune 3.x. |
+| `--new-project` | No | Name of the new project in the format `PROJECT_NAME`. Project will be created if it does not already exist. If not provided, the project name will be the same as the legacy project name. |
+| `-q`, `--query` | No | Query filter for runs to be copied ([NQL syntax](https://docs-legacy.neptune.ai/usage/nql/)). |
+| `--max-workers` | No | Maximum number of parallel workers to use for copying runs. Defaults to using ThreadPoolExecutor's default. |
+
+---
+
+## How It Works
+
+- The target project is created in the new workspace if it does not already exist.
+- All runs that match the query are copied from the legacy project to the new project.
+  - Metrics and parameters are copied from memory
+    - The `monitoring/` namespace in the source run is copied to the `runtime/` namespace in the target run.
+    - All `sys` fields (except a few) are copied to the `legacy_sys` namespace.
```

Comment:
Let's make it explicit and mention where you can see the exact ones, something like:

```
    - Most `sys` fields are copied to the `legacy_sys` namespace. For details, see [section link]
```